htmd/charge/charge.py

```diff
@@ -69,6 +70,92 @@ def fitGasteigerCharges(mol):
     return mol
 
 
+def fitChargesWithAntechamber(mol, type='gas', molCharge=None):
+    """
+    Fit atomic charges with Antechamber
+
+    Parameters
+    ----------
+    mol: Molecule
+        Molecule to fit the charges
+    type: str
+        Charge type
+    molCharge: int
+        Molecular charge
+
+    Return
+    ------
+    results: Molecule
+        Copy of the molecule with the charges set
+
+    Examples
+    --------
+    >>> from htmd.home import home
+    >>> from htmd.molecule.molecule import Molecule
+    >>> molFile = os.path.join(home('test-qm'), 'H2O.mol2')
+    >>> mol = Molecule(molFile)
+    >>> mol.charge[:] = 0
+
+    >>> new_mol = fitChargesWithAntechamber(mol)
+    >>> assert new_mol is not mol
+    >>> new_mol.charge # doctest: +ELLIPSIS
+    array([-0.411518...,  0.205759...,  0.205759...], dtype=float32)
+
+    >>> new_mol = fitChargesWithAntechamber(mol, type='gas')
+    >>> assert new_mol is not mol
+    >>> new_mol.charge # doctest: +ELLIPSIS
+    array([-0.411518...,  0.205759...,  0.205759...], dtype=float32)
+
+    >>> new_mol = fitChargesWithAntechamber(mol, type='gas', molCharge=10)
+    >>> assert new_mol is not mol
+    >>> new_mol.charge # doctest: +ELLIPSIS
+    array([-0.411518...,  0.205759...,  0.205759...], dtype=float32)
+
+    >>> new_mol = fitChargesWithAntechamber(mol, type='bcc')
+    >>> assert new_mol is not mol
+    >>> new_mol.charge # doctest: +ELLIPSIS
+    array([-0.785...,  0.39...,  0.39...], dtype=float32)
+
+    >>> new_mol = fitChargesWithAntechamber(mol, type='bcc', molCharge=0)
+    >>> assert new_mol is not mol
+    >>> new_mol.charge # doctest: +ELLIPSIS
+    array([-0.785...,  0.39...,  0.39...], dtype=float32)
+    """
+
+    if not isinstance(mol, Molecule):
+        raise TypeError('"mol" has to be instance of {}'.format(Molecule))
+    if mol.numFrames != 1:
+        raise ValueError('"mol" can have just one frame, but it has {}'.format(mol.numFrames))
+
+    if type not in ('gas', 'bcc'):
+        raise ValueError('"type" has to be "gas" or "bcc"')
+
+    if molCharge is None:
+        molCharge = int(round(np.sum(mol.charge)))
+        logger.info('Using partial atomic charges to calculate molecular charge')
```

Comment:
You could also report the total charge here maybe? Could be useful for detecting issues. Although I think cli.py is already logging the charges?

Reply:
Yes, CLI already reports